api/src/main/java/org/pmiops/workbench/config/GenderRaceEthnicityConceptCache.java

```diff
@@ -0,0 +1,65 @@
+package org.pmiops.workbench.config;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import org.pmiops.workbench.cdr.cache.GenderRaceEthnicityConcept;
+import org.pmiops.workbench.cdr.cache.GenderRaceEthnicityType;
+import org.pmiops.workbench.cdr.dao.ConceptDao;
+import org.pmiops.workbench.cdr.model.Concept;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.web.context.annotation.RequestScope;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+@Configuration
+public class GenderRaceEthnicityConceptCache {
```

Comment:
I'm not sure if we'll have cause to cache other concepts in future, but we could rename this "ConceptCache" and have MAIN_CONFIG_ID be GENDER_RACE_ETHNICITY_ID or whatever... that would make it possible to reuse the cache for other concepts.

Also we might want "Configuration" as a suffix for this class name to indicate it's a Spring config?

Reply:
Done.